docs/_documentations/remotedeploy-eclipse.md

```diff
@@ -24,14 +24,16 @@ In this topic you will:
 
 ## 1. Connect your IDE's Codewind extension to your remote instance of Codewind.
 
-1\. Locate the Codewind Explorer view in Eclipse and click the cloud icon to launch the new connection wizard:
+1\. Locate the Codewind Explorer view in Eclipse and click the cloud icon to start the new connection wizard:
 
 ![New Connection](./images/remoteeclipse/newConnection.png)
 
-2\. Add a connection name, the connection URL of your deployment, user name, and password:
+2\. Add a connection name, the connection URL of your deployment, username, and password:
```

Comment:
I've seen "user name" refer to the word but "username" refer to a variable. This is another style guide thing that we should revisit with the team. It might be best to use only one version for consistency/simplicity.

Reply:
Had a discussion with Sarah, going with username as this is what Acrolinx reports. Team to revisit style guide.